src/app/ui/components/flag/flag.tsx

```diff
@@ -21,6 +21,7 @@ export interface FlagProps extends FlexProps {
   align?: FlagAlignment;
   children: React.ReactNode;
   img?: React.ReactNode;
+  reverse?: boolean;
```

Comment:
Stole this code to use so don't duplicate efforts ...seemed like a good idea for the layout here too.

Reply:
@kyranjamie I stole this code from you thinking you would merge first, if I merge first is that ok?